package.json

```diff
@@ -61,7 +61,7 @@
     "@nypl/design-toolkit": "0.1.22",
     "@nypl/dgx-header-component": "2.0.0",
     "@nypl/dgx-react-footer": "0.4.0",
-    "dgx-alt-center": "git+https://bitbucket.org/NYPL/dgx-alt-center.git#master",
+    "@nypl/nypl-data-api-client": "^0.2.3",
```

Comment:
Is `nypl-data-api-client` doing the same work as `dgx-alt-center`? Or it just we don't `dgx-alt-center`.